INTERNALS.md

```diff
@@ -0,0 +1,32 @@
+# Architecture
+
+The document [Alternative Broker Implementation based on Apache Kafka](https://docs.google.com/document/d/10-qylWrj7Tj81EqoIiZ2AANAZNmkKe7XKmFobgZULKY/edit)
+explains the architecture, and some reasons for implementing a native Kafka Broker instead of using a channel-based Broker. 
+(You need to join `knative-dev` group to view it).
```

Comment:
More than linking the doc, could you report here the major things explained in the doc?

Reply:
Can we do this when we consolidate the implementation after we implement the `control-plane`?

Reply:
Sure, open an issue for that :wink:

Reply:
created: https://github.com/knative-sandbox/eventing-kafka-broker/issues/13